cmd/kube-scheduler/app/options/deprecated.go

```diff
@@ -126,14 +126,12 @@ func (o *DeprecatedOptions) ApplyTo(cfg *kubeschedulerconfig.KubeSchedulerConfig
 	if len(o.SchedulerName) > 0 {
 		profile.SchedulerName = o.SchedulerName
 	}
-	if o.HardPodAffinitySymmetricWeight != interpodaffinity.DefaultHardPodAffinityWeight {
-		plCfg := kubeschedulerconfig.PluginConfig{
-			Name: interpodaffinity.Name,
-			Args: &kubeschedulerconfig.InterPodAffinityArgs{
-				HardPodAffinityWeight: o.HardPodAffinitySymmetricWeight,
-			},
-		}
-		profile.PluginConfig = append(profile.PluginConfig, plCfg)
+	plCfg := kubeschedulerconfig.PluginConfig{
```

Comment:
I would update the comment in line 123 to say something like: "deprecated flags have an effect iff no config file was provided, in which case this function expects a default CC instance with a single default profile".